spec/classes/foreman_proxy_content_spec.rb

```diff
@@ -12,12 +12,6 @@
       end
 
       context 'with pulpcore' do
```

Comment:
This now becomes the same as `without parameters` so we can merge those two groups. https://github.com/theforeman/puppet-foreman_proxy_content/pull/325